doc/user/writing_spec.md

```diff
@@ -0,0 +1,45 @@
+# Writing Cluster Spec
+ 
+Users tell operator the desired state of an etcd cluster via spec in `Cluster` third party resource (TPR).
+See [example](../../example/example-etcd-cluster.yaml).
+
+Users might wonder what to specify in the spec.
```

Comment:
I feel this is better than what we have here at least. Kubernetes doc uses markdown table too. You can search around to see if there is a utility that can render a markdown table from go code. We render proto message to markdown table for etcd.